gn3/settings.py

```diff
@@ -27,3 +27,21 @@
 
 # wgcna script
 WGCNA_RSCRIPT = "wgcna_analysis.R"
+# qtlreaper command
+REAPER_COMMAND = "{}/bin/qtlreaper".format(os.environ.get("GUIX_ENVIRONMENT"))
```

Comment:
**Nitpick:** Prefer f-strings to format-string. Only decent-ish reason I can provide is I'd like us to be more consistent with our string formatting.